operations/services.md

```diff
@@ -1089,8 +1106,8 @@ Returns prices of a rate in the specified interval. Note that response contains
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `RateId` | string | required | Unique identifier of the [Rate](services.md#rate) whose prices should be returned. |
-| `StartUtc` | string | required, max length 3 months | Start of the interval in UTC timezone in ISO 8601 format. |
-| `EndUtc` | string | required, max length 3 months | End of the interval in UTC timezone in ISO 8601 format. |
+| `FirstTimeUnit` | [TimeUnit](#time-unit) | required, [max length](#time-unit-interval-length-restrictions) | First TimeUnit of the interval. |
+| `LastTimeUnit` | [TimeUnit](#time-unit) | required, [max length](#time-unit-interval-length-restrictions) | Last TimeUnit of the interval. |
```

Comment:
Similar here, the current contract wasn't described in a best way.

Reply:
Added the info about that to paragraph describing the max length.